packages/docs/components/text-input.md

```diff
@@ -173,6 +173,77 @@ Note, when indicating a validation error, please use a `.ods-field--error` to in
 
 ## Additional types
 
+### Standalone Search
```

Comment:
We plan on making this a full component later, but wanted to rely on `input[type="search"]` with a limited use case for now. Once we start adding things like autocompletion and results previews, this will def get elevated!